### PR TITLE
TC_03.04.006 | Create Pipeline project

### PR DESCRIPTION
**Pull request by JackBlaaack**
https://trello.com/c/vcGrZt5S/134-tc0304006-create-pipeline-project

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "cypress": "^12.11.0"
+    "cypress": "^12.17.4"
```
Comment (Rom-T) [inline package.json:12]: Why did you change the Cypress version? You can't do this!

**Review by Rom-T — CHANGES_REQUESTED** [review]
Why did you change the Cypress version? You can't do this!

File: package.json (hunk 1 above)
Comment (JackBlaaack) [reply package.json:12]: And how do I downgrade now?

**Review by Rom-T — APPROVED** [review]

**Review by Nataly-Mayatskaya — APPROVED** [review]
